Formula/chronograf.rb

```diff
@@ -0,0 +1,102 @@
+require "language/node"
+
+class Chronograf < Formula
+  desc "Open source monitoring and visualization UI for the TICK stack"
+  homepage "https://www.influxdata.com/time-series-platform/chronograf/"
+  url "https://github.com/influxdata/chronograf.git",
+      :tag => "1.1.0-beta2",
```

Comment:
Fine to wait on this. Do you remember the test failure? Additionally, does it reproduce locally with `brew install` and `brew test`.